modules/tinymce/src/plugins/table/test/ts/browser/ui/TableDialogStyleWithCssTest.ts

```diff
@@ -223,6 +235,18 @@ describe('browser.tinymce.plugins.table.ui.TableCellDialogStyleWithCssTest', ()
           assertTable(editor, { cellSpacingAttr: '5', cellSpacingStyle: '' });
         }
       });
+
+      it('TINY-10308: Setting table border to 0px should remove border-width styles and set border="0"', async () => {
+        const editor = hook.editor();
+        initializeTable(editor, {});
```

Comment:
I think just having `initializeTable(editor, {});` won't add any `border-width` styles to cell so we aren't properly testing anything here